kroxylicious/src/main/java/io/kroxylicious/proxy/internal/codec/KafkaRequestDecoder.java

```diff
@@ -85,12 +89,19 @@ protected Frame decodeHeaderAndBody(ChannelHandlerContext ctx, ByteBuf in, final
 
         RequestHeaderData header = null;
         final ByteBufAccessorImpl accessor;
+        Metrics.globalRegistry.counter("inbound_downstream_messages").increment();
         var decodeRequest = decodePredicate.shouldDecodeRequest(apiKey, apiVersion);
         LOGGER.debug("Decode {}/v{} request? {}, Predicate {} ", apiKey, apiVersion, decodeRequest, decodePredicate);
         boolean decodeResponse = decodePredicate.shouldDecodeResponse(apiKey, apiVersion);
         LOGGER.debug("Decode {}/v{} response? {}, Predicate {}", apiKey, apiVersion, decodeResponse, decodePredicate);
         short headerVersion = apiKey.requestHeaderVersion(apiVersion);
         if (decodeRequest) {
+            Metrics.globalRegistry.counter("decoded_messages").increment();
+            Metrics.globalRegistry.counter("bytes_read").increment(length);
```

Comment:
We don't need this counter as the DistributionSummary will provide it for us anyway see the tip in the [counter docs](https://micrometer.io/docs/concepts#_counters)